DEPLOYING.md

```diff
@@ -0,0 +1,51 @@
+# Deploying and Development
+
+The VSCode Extension for PRQL relies on the `prqlc` code from the
+main PRQL repo. There are two Github Actions that wacth for updates
+in this directory to build and release the extension to the
+Microsoft Marketplace.
+
+## Deploying/Updating the Extension
+
+When there is a new version of PRQL (`prqlc` in particular):
+
+- In _package.json_, update:
+
+  - `version` to the proper version (from the main PRQL repo)
+  - `dependencies.prqlc` to the same version
+  - (optional) update other dependencies and devDependencies
```

Comment:
```suggestion
- In [package.json](package.json), update `version` to the matching version from the main PRQL repo.
```

Reply:
There are actually two `version`s that I assume should be the same. True?
- right near the top of the JSON object (for the extension's version itself, I imagine)
- in `dependencies`, there's a `prqlc` property that tells which version to include